x/axelarnet/keeper/migrate.go

```diff
@@ -1,20 +1,35 @@
 package keeper
 
 import (
+	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/axelarnetwork/axelar-core/x/axelarnet/types"
 )
 
-// Migrate5to6 returns the handler that performs in-place store migrations from version 5 to 6
-func Migrate5to6(k Keeper) func(ctx sdk.Context) error {
+// Migrate6to7 returns the handler that performs in-place store migrations from version 6 to 7
+func Migrate6to7(k Keeper, bankK types.BankKeeper, accountK types.AccountKeeper, nexusK types.Nexus, ibcK IBCKeeper) func(ctx sdk.Context) error {
+
 	return func(ctx sdk.Context) error {
-		addModuleParamCallContractsProposalMinDeposits(ctx, k)
+		// Failed IBC transfers are held in Axelarnet module account for later retry.
+		// This migration escrows tokens back to escrow accounts so that we can use the same code path for retry.
+		moduleAccount := accountK.GetModuleAddress(types.ModuleName)
+
+		balances := bankK.SpendableCoins(ctx, moduleAccount)
+		for _, coin := range balances {
+			asset, err := nexusK.NewLockableAsset(ctx, ibcK, bankK, coin)
+			if err != nil {
+				k.Logger(ctx).Info(fmt.Sprintf("coin %s is not a lockable asset", coin), "error", err)
+				continue
+			}
+
+			err = asset.LockFrom(ctx, moduleAccount)
```

Comment:
app.go now only gives send permission to `nexus` module account. So locking from axelarnet module on upgrade will fail since it's not [whitelisted](https://github.com/axelarnetwork/axelar-core/blob/27666e016047822917cc45ff9192aab422a7b199/app/keepers.go#L451) from being a blocked address. Alternatively, we could remove it from the module [map](https://github.com/axelarnetwork/axelar-core/blob/27666e016047822917cc45ff9192aab422a7b199/app/app.go#L720) perhaps?

Reply:
Alternatively, could use `SendCoinsFromModuleToModule` to send to nexus module and lock from there. Seems like that should skip the blocked address check?